### DISCUSSION TITLE
API Error in scraping Metrics

### DISCUSSION THREAD

**Discussion by sumant5476** [tomkerkhove/promitor#2077] [Q&A]
Hi @tomkerkhove,

<img width="752" alt="pture" src="https://user-images.githubusercontent.com/71654461/179165837-b68e4d14-bf8c-4d1a-896e-f63b2cbab361.PNG">

Getting this error not sure why can you help?

Thanks in Advance!

**Comment by tomkerkhove** [comment]
I don't have enough information fort that though; did you check the logs of the resource discovery agent?
